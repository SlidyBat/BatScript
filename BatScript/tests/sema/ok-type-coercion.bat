def g(): return 3

float f = 10
float f2 = 10 + 5.1
float f3 = g()

print f
print f2
print f3