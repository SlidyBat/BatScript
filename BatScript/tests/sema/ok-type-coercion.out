10.000000
15.100000
3.000000
